source/backend/api/Services/PropertyService.cs

```diff
@@ -209,9 +209,70 @@ public PropertyManagementModel GetPropertyManagement(long propertyId)
             propertyManagement.RelatedLeases = leaseCount;
             propertyManagement.LeaseExpiryDate = leaseExpiryDate.HasValue ? DateOnly.FromDateTime(leaseExpiryDate.Value) : null;
 
+            PropertyHasActiveLease(propertyLeases, out bool hasActiveLease, out bool hasActiveExpiryDate);
+            propertyManagement.HasActiveLease = hasActiveLease;
+            propertyManagement.ActiveLeaseHasExpiryDate = hasActiveExpiryDate;
+
             return propertyManagement;
         }
 
+        private static void PropertyHasActiveLease(IEnumerable<PimsPropertyLease> propertyLeases, out bool hasActiveLease, out bool hasActiveExpiryDate)
+        {
+            hasActiveLease = false;
+            hasActiveExpiryDate = false;
+
+            List<PimsLease> activeAgreementsList = propertyLeases.Select(x => x.Lease).ToList();
+            var activeLease = activeAgreementsList.FirstOrDefault(x => x.LeaseStatusTypeCode == LeaseStatusTypes.ACTIVE.ToString());
+
+            // No Active Lease
+            if (activeLease is not null)
+            {
+                // Lease is Active but has termination Date.
+                if (activeLease.TerminationDate.HasValue)
+                {
+                    return;
+                }
+                else
+                {
+                    var latestRemewal = activeLease.PimsLeaseRenewals.Where(x => x.IsExercised.HasValue && x.IsExercised.Value).OrderByDescending(x => x.CommencementDt).FirstOrDefault();
```

Comment:
nit latestRemewal -> latestRenewal

Reply:
FYI a fast way of checking if a nullable boolean is a specific value is just x.IsExercised == true, since null != true and false != true